dataprofiler/tests/profilers/test_datetime_column_profile.py

```diff
@@ -371,6 +373,25 @@ def test_datetime_column_with_wrong_options(self):
                                    " of type DateTimeOptions."):
             profiler = DateTimeColumn("Datetime", options="wrong_data_type")
 
+    def test_day_suffixes(self):
+        """
+        Tests datetime examples with daytime suffixes.
+        :return:
+        """
+        data = [
+            'Mar 1st, 2020',
+            'Feb 22nd, 2019',
+            'October 23rd, 2018',
+            '12thMar13'
+        ]
```

Comment:
ideally we should check to make sure each is properly updated. This could technically be done by checking `samples` maybe?

Reply:
`profiler.sample`

Reply:
rethinking that it is recognizing all 4 as dates since `match_count`